INSTALLATION.rst

```diff
@@ -85,7 +88,7 @@ Recommended Method
 
      and for Python 3.x, you should be able to do::
 
-      conda create --name <MYFIPYENV> --channel conda-forge python=3.6 numpy scipy matplotlib pysparse
+      conda create --name <MYFIPYENV> --channel conda-forge "python>3" numpy scipy matplotlib pysparse gmsh
```

Comment:
is `"python>3"` better than `python=3`. I guess that works when python goes to version 4 as well.

Reply:
I didn't test. I worried that might give me Python 3.0. We probably *don't* want it to start using Python 4 without warning us.

Reply:
I use `python=3` generally to specify the latest version that conda is using.

Reply:
with circle ci or just installing locally

Reply:
Thanks